query-engine/prisma-models/src/field_selection.rs

```diff
@@ -77,7 +77,7 @@ impl FieldSelection {
                 SelectedField::Scalar(sf) => Some(sf.clone()),
                 SelectedField::Composite(_) => None,
             })
-            .collect_vec();
+            .collect::<Vec<_>>();
```

Comment:
🐟

Reply:
Any reason why we wouldn't wanna use `Itertools::collect_vec()` ? 🤔

Reply:
I have a (partially irrational) bias against itertools.